src/main/services/request-auth.ts

```diff
@@ -64,10 +68,11 @@ export class AutomationCloudAuthService extends RequestAuthService {
         }
         try {
             const jwt = await this.jwt.decodeAndVerify(token);
+            this.validateJwt(jwt);
             this.acContext.set({
                 authenticated: true,
-                organisationId: jwt.context?.organisation_id ?? null,
-                // TODO extract more stuff
+                organisationId: jwt.context.organisation_id ?? null,
+                jwt,
```

Comment:
Idk about putting raw JWT data into the context. Too many parties know about JWT (as a transport) this way.

Reply:
yup I understand your concern 💯 , I didn't have much choice for now as our auth is a bit immature and we don't really know what information is needed for each application to control the requests, apart from organisation info (distinguishing User ServiceUser Job etc..) so as a starter I put all the info we've got in there